adapters/adnuntius/params_test.go

```diff
@@ -41,6 +41,7 @@ func TestInvalidParams(t *testing.T) {
 var validParams = []string{
 	`{"auId":"123"}`,
 	`{"auId":"123", "network":"test"}`,
+	`{"auId":"123", "network":"test", "nativeRequest":{}}`,
```

Comment:
This test is for `imp.ext.bidder` params only. They are specified in `adnuntius.json` and `imp_adnuntuis.go`.

`nativeRequest` doesn't look like a new field you want to support. Please remove this line.